packages/manager/apps/dedicated/client/app/dedicatedCloud/datacenter/backup/translations/Messages_fr_FR.json

```diff
@@ -10,6 +10,8 @@
   "datacenter_backup_capability_diskSizeInReport": "Taille du disque",
   "datacenter_backup_capability_fullDayInReport": "Sauvegarde complète journalière",
   "datacenter_backup_capability_restorePointInReport": "Restaurer mes points",
-  "datacenter_backup_capability_mailAddress": "Envoyer le rapport journalier de sauvegarde à une adresse e-mail spécifique",
-  "datacenter_backup_capability_update": "Valider les options du rapport"
+  "datacenter_backup_capability_sendReportToCustomeEmail": "Envoyer le rapport journalier de sauvegarde à une adresse e-mail spécifique",
+  "datacenter_backup_capability_update": "Valider les options du rapport",
+  "datacenter_backup_capability_update_error": "Unable to update backup capabilities: {{ message }}",
```

Comment:
`Une erreur est survenue lors de la mise à jour de vos paramètres de sauvegarde : {{ message }}`

in English

`An error occurred while updating your backup settings. `